apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +255,118 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  courseShortName,
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+  timezone,
+}: {
+  courseShortName: string;
+  csrfToken: string;
+  initialStartDateFormatted: string;
+  initialEndDateFormatted: string;
+  timezone: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create course instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="long_name">Long Name</label>
+        <input
+          type="text"
+          class="form-control"
+          id="long_name"
+          name="long_name"
+          required
+          aria-describedby="long_name_help"
+        />
+        <small id="long_name_help" class="form-text text-muted">
+          The full course instance name, such as "Fall 2019". Users see it joined to the course
+          name, e.g. "${courseShortName} Fall 2019".
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="short_name">Course Instance Identifier (CIID)</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          aria-describedby="short_name_help"
+        />
+        <small id="short_name_help" class="form-text text-muted">
+          ID for internal use, of format
+          <code>Fa19</code> or <code>Fall2019</code>. Use only letters, numbers, dashes, and
+          underscores, with no spaces.
+        </small>
+      </div>
+      <div class="form-check mb-3">
+        <input
+          type="checkbox"
+          class="form-check-input"
+          id="access_dates_enabled"
+          name="access_dates_enabled"
+          aria-describedby="access_dates_enabled_help"
+        />
+        <label class="form-check-label" for="access_dates_enabled"
+          >Make course instance available to students</label
+        >
+        <small id="access_dates_enabled_help" class="form-text text-muted">
+          This can be enabled later.
+        </small>
+      </div>
+      <div id="accessDates" hidden="true">
+        <div class="form-group">
+          <label for="start_access_date">Start Access Date</label>
```

Comment:
Nit, this is how we usually write this (you can change the end date too, and probably the input IDs and names as well):

```suggestion
          <label for="start_access_date">Access Start Date</label>
```